docs/core/Mercury.md

```diff
@@ -0,0 +1,346 @@
+      },
+      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
+      "latestConfig": {
+        "offchainConfig": {
+          "deltaGrace": "0",
+          "deltaProgress": "2s",
+          "deltaResend": "20s",
+          "deltaRound": "250ms",
+          "deltaStage": "60s",
+          "f": 1,
+          "maxDurationObservation": "250ms",
+          "maxDurationQuery": "0s",
+          "maxDurationReport": "250ms",
+          "maxDurationShouldAcceptFinalizedReport": "250ms",
+          "maxDurationShouldTransmitAcceptedReport": "250ms",
+          "rMax": 100,
+          "reportingPluginConfig": {
+            "alphaAcceptInfinite": false,
+            "alphaAcceptPpb": "0",
+            "alphaReportInfinite": false,
+            "alphaReportPpb": "0",
+            "deltaC": "0s"
+          },
+          "s": [
+            4
+          ]
+        },
+        "offchainConfigVersion": 30,
+        "onchainConfig": {
+          "max": "99999999999999999999999999999",
+          "min": "1"
+        },
+        "onchainConfigVersion": 1,
+        "oracles": [
+          {
+            "api": [
+              "coinmetrics",
+              "ncfx",
+              "tiingo-test"
+            ],
+            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-0"
+          },
+          {
+            "api": [
+              "coinmetrics",
+              "ncfx",
+              "tiingo-test"
+            ],
+            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-1"
+          },
+          {
+            "api": [
+              "coinmetrics",
+              "ncfx",
+              "tiingo-test"
+            ],
+            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-2"
+          },
+          {
+            "api": [
+              "coinmetrics",
+              "ncfx",
+              "tiingo-test"
+            ],
+            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-3"
+          }
+        ]
+      },
+      "marketing": {
+        "category": "crypto",
+        "history": true,
+        "pair": [
+          "LINK",
+          "USD"
+        ],
+        "path": "link-usd-verifier"
+      },
+      "name": "LINK/USD-RefPricePlus-ArbitrumGoerli-002",
+      "reportFields": {
+        "ask": {
+          "decimals": 8,
+          "maxSubmissionValue": "99999999999999999999999999999",
+          "minSubmissionValue": "1",
+          "resultPath": "data,ask"
+        },
+        "bid": {
+          "decimals": 8,
+          "maxSubmissionValue": "99999999999999999999999999999",
+          "minSubmissionValue": "1",
+          "resultPath": "data,bid"
+        },
+        "median": {
+          "decimals": 8,
+          "maxSubmissionValue": "99999999999999999999999999999",
+          "minSubmissionValue": "1",
+          "resultPath": "data,mid"
+        }
+      },
+      "status": "testing"
+    }
+  },
+  "name": "Mercury v0.2 - Production Testnet Verifier (v1.0.0)",
+  "status": "testing"
+}
+```
```

Comment:
You could use `<details><summary>` sections to make examples collapsible. I don't know how to suggest with the nested `, but you should be able to 'edit' this comment to see the source I used:

<details><summary>Example `verifier/<0xaddress>.json`:</summary>

```json
{
  "contractVersion": 1001,
  "digests": {
    "0x0006c67c0374ab0dcfa45c63b37df2ea8d16fb903c043caa98065033e9c15666": {
      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
      "proxyEnabled": true,
      "status": "active"
    }
  },
  "feeds": {
    "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c": {
      "digests": [
        "0x0006c67c0374ab0dcfa45c63b37df2ea8d16fb903c043caa98065033e9c15666"
      ],
      "docs": {
        "assetName": "Chainlink",
        "feedCategory": "verified",
        "feedType": "Crypto",
        "hidden": true
      },
      "externalAdapterRequestParams": {
        "endpoint": "cryptolwba",
        "from": "LINK",
        "to": "USD"
      },
      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
      "latestConfig": {
        "offchainConfig": {
          "deltaGrace": "0",
          "deltaProgress": "2s",
          "deltaResend": "20s",
          "deltaRound": "250ms",
          "deltaStage": "60s",
          "f": 1,
          "maxDurationObservation": "250ms",
          "maxDurationQuery": "0s",
          "maxDurationReport": "250ms",
          "maxDurationShouldAcceptFinalizedReport": "250ms",
          "maxDurationShouldTransmitAcceptedReport": "250ms",
          "rMax": 100,
          "reportingPluginConfig": {
            "alphaAcceptInfinite": false,
            "alphaAcceptPpb": "0",
            "alphaReportInfinite": false,
            "alphaReportPpb": "0",
            "deltaC": "0s"
          },
          "s": [
            4
          ]
        },
        "offchainConfigVersion": 30,
        "onchainConfig": {
          "max": "99999999999999999999999999999",
          "min": "1"
        },
        "onchainConfigVersion": 1,
        "oracles": [
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-0"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-1"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-2"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-3"
          }
        ]
      },
      "marketing": {
        "category": "crypto",
        "history": true,
        "pair": [
          "LINK",
          "USD"
        ],
        "path": "link-usd-verifier"
      },
      "name": "LINK/USD-RefPricePlus-ArbitrumGoerli-002",
      "reportFields": {
        "ask": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,ask"
        },
        "bid": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,bid"
        },
        "median": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,mid"
        }
      },
      "status": "testing"
    }
  },
  "name": "Mercury v0.2 - Production Testnet Verifier (v1.0.0)",
  "status": "testing"
}
```
</details>